e2e/features/appeal-application.feature

```diff
@@ -10,7 +10,11 @@ Feature: Complete the appeal application online
     Then I should see the Basic details page
 
     Given I am on the Basic details page
-    When I add a new item to the Nationality collection
+    Then I type Mr for the Title field
```

Comment:
These lines could've been "When"s instead of "Then"s for clarity, but doesn't affect the test itself

Reply:
Quite right -- I will adjust the When/Then in a later PR that way I will catch them all.